daffodil-cli/src/it/scala/org/apache/daffodil/blob/TestBlob.scala

```diff
@@ -82,40 +69,41 @@ class TestBlob {
     }
   }
 
-  @Test def test_1GB_blob() {
+  /* These large file tests are commented out so that they are not triggered on
+   * automatic regression tests on the build servers.  In order to run them you
+   * will need to unzip the file:
+   *
+   * daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/2_01GB.zip
+   *
+   * and make sure that the file "2_01GB.bin" ends up in:
+   *
+   * daffodil-cli/src/it/resources/org/apache/daffodil/CLI/input/
+   */
+  /*@Test def test_2GB_blob() {
```

Comment:
You can comment out a test by just commenting out the Test marker
`/* @Test */
def thisTestCommentedOut() {....
}`

Reply:
I actually kindof prefer commenting out the whole thing. It makes it much more obvious that a test isn't going to be run when the whole thing is syntax highlighted as a comment rather than just the one ``@Test`` annotation.